churchslavonic.tex

```diff
@@ -102,7 +103,7 @@ \section*{Введение}
 отдельных утилит, которые можно найти на сайте
 \href{http://sci.ponomar.net/}
 {Инициативной группы славянской информатики}.
-Для того, чтобы использовать возможности этого пакета Вам
+Для того, чтобы использовать возможности этого пакета Вам
```

Comment:
strange diff. blank replaced with blank???

Reply:
It's was a U+00A0 No-Break Space. I replaced it with a usual space.